tests/utils.py

```diff
@@ -288,3 +289,19 @@ def test_thug_url(self, mock_get=None, mock_post=None):
             additional_params,
         ).start()
         self.assertEqual(report.get("success", False), True)
+
+class CommonTestCases_generic(metaclass=ABCMeta):
```

Comment:
move this class to `test_observables.py` file with name like `class GenericAnalyzersTest(TestCase):`

Reply:
also add this at the top of this new class,

```python
    @staticmethod
    def get_params():
        return {
            "source": "test",
            "is_sample": False,
            "observable_name": os.environ.get(
                "TEST_GENERIC", "email@example.com"
            ),
            "observable_classification": "generic",
            "force_privacy": False,
            "analyzers_requested": ["test"],
        }

```

Reply:
done